serializations.md

```diff
@@ -121,6 +141,8 @@ subsequent sections divided by object.
 | Channel settle transaction | 56 |
 | Channel off chain transaction | 57 |
 | Channel | 58 |
+| Key block | 100 |
+| Micro block | 101 |
```

Comment:
Not needed as it only makes sense for serialization as `S(O) = rlp([tag(O), vsn(O)] ++ fields(O))` while serialization is defined as byte array.